webapp/src/generalSlices/configSlice.ts

```diff
@@ -37,32 +37,6 @@ export const refreshSnmpFwdConfig = createAsyncThunk(
   }
 )
 
-export const refreshSnmpFwdConfigManual = createAsyncThunk(
-  'config/refreshSnmpFwdConfigManual',
-  async (ipList: string[], { getState, dispatch }) => {
-    const currentState = getState() as RootState
-    const token = selectToken(currentState)
-    const organization = selectOrganizationName(currentState)
-
-    const body: RsuCommandPostBody = {
-      command: 'rsufwdsnmpwalk',
-      rsu_ip: ipList,
-      args: {},
-    }
-
-    const response = await RsuApi.postRsuData(token, organization, body, '')
-
-    return response.status === 200
-      ? { msgFwdConfig: response.body.RsuFwdSnmpwalk, errorState: '' }
-      : {
-          msgFwdConfig: {} as {
-            [id: string]: SnmpFwdWalkConfig
-          },
-          errorState: response.body.RsuFwdSnmpwalk,
-        }
-  }
-)
-
```

Comment:
Removed this function from the webapp since it was entirely unused from an older implementation.